packages/react/src/FronteggProvider.tsx

```diff
@@ -30,14 +31,19 @@ export const Connector: FC<ConnectorProps> = ({ history, appName, ...props }) =>
 
   // v6 or v5
   const baseName = props.basename ?? '';
+  const { authenticatedUrl, ...authRoutes } = { ...authInitialState.routes, ...props.authOptions?.routes };
+  const isAuthRouteRef = useRef<(path: string) => boolean>(() => false);
+  useMemo(() => {
```

Comment:
correct me if I'm wrong, but if u don't use the return value from the `useMemo` u can just use `useEffect`